src/containers/AddCardForm.js

```diff
@@ -0,0 +1,11 @@
+import { connect } from "react-redux";
+import { addCard } from "../actions";
+import AddCardForm from "../components/AddCard/AddCardForm/AddCardForm";
+
+
+const mapDispatchToProps = dispatch => ({
```

Comment:
There is shorthand notation that connects dispatch automatically for you.

```
const mapDispatchToProps = dispatch: => {
  handleAddCard: addCard,
}
```
But then you need to pass proper params in component `this.props.handleAddCard(card.name, card.email)` or accept the whole card and extract needed data in action creator (using destructuring)
`export const addCard = ({ name, email }) => ({`

Reply:
Thank you I used it